src/BigBlueButton.php

```diff
@@ -470,6 +511,10 @@ public function setJSessionId(string $jSessionId): void
     }
 
     /**
+     * Sets curl options.
+     *
+     * This has no effect if the instance has a http client.
```

Comment:
I think this would be "an http client"..
```suggestion
     * This has no effect if the instance has an http client.
```
and same for other similar places.